pkgs/development/libraries/openssl/default.nix

```diff
@@ -102,7 +114,17 @@ let
     configureFlags = [
       "shared" # "shared" builds both shared and static libraries
       "--libdir=lib"
-      "--openssldir=etc/ssl"
+      (if !static then
+         "--openssldir=etc/ssl"
+       else
+         # Separate the OPENSSLDIR into its own output, as its path will be
+         # compiled into 'libcrypto.a'. This makes it a runtime dependency of
+         # any package that statically links openssl, so we want to keep that
+         # output minimal. We need to prepend '/.' to the path in order to make
+         # it appear absolute before variable expansion, the 'prefix' would be
+         # prepended to it otherwise.
```

Comment:
I think this text belongs to definition of the `outputs` attribute, where the "etc" output is first declared. This place is just an implementation detail.

Reply:
I've tried to address this, does this make sense?

(It's a bit tricky because I don't really understand what's going on here.)

Reply:
Happy to cut the comments down further, until they don't say anything misleading :D.